app/controllers/trips_controller.rb

```diff
@@ -35,8 +37,8 @@ def create
       end
       @trip_activity = TripActivity.create(trip: @trip, activity: @activity)
     end
-    raise
-    # add download and upload map_image
+    @map_image = Down.download("https://api.mapbox.com/styles/v1/mapbox/outdoors-v11/static/pin-s+601d72(#{@trip.longitude},#{@trip.latitude})/#{@trip.longitude},#{@trip.latitude},7,0/300x300?access_token=pk.eyJ1IjoiZmRlc21hcmFpcyIsImEiOiJja216NXpjcTgwNzNsMm5sYzhqcmlzcHFhIn0.Aus1LPrDbVcjE-1wx8qwGQ")
```

Comment:
Careful not to expose the access token. :S I'll need to generate a new key since this repo is public. 
No biggy ;)